domain/src/commonMain/kotlin/com/islamversity/domain/mapper/SurahMapper.kt

```diff
@@ -0,0 +1,10 @@
+package com.islamversity.domain.mapper
+
+import com.islamversity.core.Mapper
+import com.islamversity.db.model.Surah
+import com.islamversity.domain.model.SurahUIModel
+
+class SurahMapper : Mapper<Surah, SurahUIModel> {
```

Comment:
please consider using `SurahDBUIMapper` this way we know what it does more by looking at the name